graphql/executor/executor.go

```diff
@@ -29,11 +31,12 @@ var _ graphql.GraphExecutor = &Executor{}
 // recovery callbacks, and no query cache or extensions.
 func New(es graphql.ExecutableSchema) *Executor {
 	e := &Executor{
-		es:             es,
-		errorPresenter: graphql.DefaultErrorPresenter,
-		recoverFunc:    graphql.DefaultRecover,
-		queryCache:     graphql.NoCache{},
-		ext:            processExtensions(nil),
+		es:               es,
+		errorPresenter:   graphql.DefaultErrorPresenter,
+		recoverFunc:      graphql.DefaultRecover,
+		queryCache:       graphql.NoCache{},
+		ext:              processExtensions(nil),
+		parserTokenLimit: 0, // 0 means unlimited
```

Comment:
We can define a constant `const parserTokenNoLimit = 0` and remove the comment:

```suggestion
		parserTokenLimit: parserTokenNoLimit,
```